src/invidious/yt_backend/extractors.cr

```diff
@@ -210,6 +211,56 @@ private module Parsers
     end
   end
 
+  # Parses an Innertube `hashtagTileRenderer` into a `SearchHashtag`.
+  # Returns `nil` when the given object is not a `hashtagTileRenderer`.
+  #
+  # A `hashtagTileRenderer` is a kind of search result.
+  # It can be found when searching for any hashtag (e.g "#hi" or "#shorts")
+  module HashtagRendererParser
+    def self.process(item : JSON::Any, author_fallback : AuthorFallback)
+      if item_contents = item["hashtagTileRenderer"]?
+        return self.parse(item_contents)
+      end
+    end
+
+    private def self.parse(item_contents)
+      text = item_contents.dig("hashtag", "simpleText").as_s # E.g "#hi"
```

Comment:
Should `extract_text` be used instead of checking the simpleText?

Reply:
You're right, thanks!